packages/react-native-windows-init/README.md

```diff
@@ -15,9 +15,14 @@ npx react-native-windows-init --language cpp
 Options:
```

Comment:
This table is also published in the main documentation at https://microsoft.github.io/react-native-windows/docs/getting-started , so updating https://github.com/microsoft/react-native-windows-samples/blob/master/docs/getting-started.md with the new telemetry information would probably be a good idea too.

Reply:
thanks! yep, the website is on a separate repo so i'll update that once this goes in

Reply:
https://github.com/microsoft/react-native-windows-samples/pull/310